src/js/components/dashboard/DashboardContent.jsx

```diff
@@ -10,178 +10,199 @@ import DashboardFilters from './DashboardFilters';
-                            page={this.state.activePage}
-                            session={this.props.session}
-                            type={this.props.type} />
-                    </div>
-                </div>
-                <div className="row">
-                    <div className="col-md-12">
-                        <div className="table-heading">
-                            <h2 className="table-heading__title">{this.state.title}</h2>
-                            {secondMessage}
-                        </div>
-                        <DashboardFilters
-                            toggleFilter={this.toggleFilter}
-                            stagedFilters={stagedFilters[secondTable]}
-                            appliedFilters={appliedFilters[secondTable]}
-                            table={secondTable}
-                            type={this.props.type} />
-                        <FilterBarContainer
-                            type={this.props.type}
-                            table={secondTable}
-                            stagedFilters={stagedFilters[secondTable]}
-                            appliedFilters={appliedFilters[secondTable]}
-                            updateFilterCount={this.updateFilterCount} />
-                        <DashboardTable
-                            isLoading={this.props.certifiedLoading}
-                            loadTableData={this.props.loadTableData}
-                            appliedFilters={appliedFilters[secondTable]}
-                            total={this.props.certifiedTotal}
-                            data={this.props.certifiedSubmissions}
-                            page={this.state.certifiedPage}
-                            session={this.props.session}
-                            type={this.props.type} />
-                    </div>
-                </div>
+    return null;
+  }
+
+  render() {
+    const stagedFilters = this.props.stagedFilters[this.props.type];
+    const appliedFilters = this.props.appliedFilters[this.props.type];
+    const secondTable = `${this.props.type === 'fabs' ? 'published' : 'certified'}`;
+
+    const activeMessage = this.generateMessage(this.state.filterCounts[this.props.type].active);
+    const secondMessage = this.generateMessage(this.state.filterCounts[this.props.type][secondTable]);
+
+    return (
+      <div className="container">
+        <div className="row">
+          <div className="col-md-12">
+            <div className="table-heading">
+              <h2 className="table-heading__title">Active Submissions</h2>
+              {activeMessage}
             </div>
-        );
-    }
+            <DashboardFilters
+              toggleFilter={this.toggleFilter}
+              bubbledRemovedFilterValue={this.state.bubbledRemovedFilterValue}
+              stagedFilters={stagedFilters.active}
+              appliedFilters={appliedFilters.active}
+              type={this.props.type}
+              table="active"
+            />
```

Comment:
Another minor code style difference - could you put `/>` on the same line as the last prop (with a space between) for consistency?